src/entity/posts/Post.ts

```diff
@@ -38,7 +38,7 @@ export type PostFlags = Partial<{
   private: boolean;
   visible: boolean;
   showOnFeed: boolean;
-  promoteToPublic: number;
+  promoteToPublic: number | null;
```

Comment:
You can't unsed jsonb value with `undefined` so we set null (its already happening this is just the type update)